chroma-manager/tests/framework/integration/existing_filesystem_configuration/cluster_setup

```diff
@@ -75,7 +75,12 @@ if $JENKINS; then
 else
     ARCHIVE_PATH=chroma-bundles
 fi
-scp $ARCHIVE_PATH/$ARCHIVE_NAME $CHROMA_DIR/chroma-manager/tests/utils/install.exp root@$CHROMA_MANAGER:/tmp
+
+if [ -f ~/storage_server.repo ]; then
```

Comment:
How does this get there?

Reply:
A developer/test infrastructure administrator puts it there.

This the hook that allows a given site to override where repos are.  So, for Lotus, it could be used to point at it's Cobbler server instead of using repos out on the Internet -- with all of their inherent [in-]stability issues.